flink-test-utils-parent/flink-test-utils/src/main/java/org/apache/flink/test/util/SecureTestEnvironment.java

```diff
@@ -37,9 +37,39 @@
 /**
  * Helper {@link SecureTestEnvironment} to handle MiniKDC lifecycle.
  * This class can be used to start/stop MiniKDC and create secure configurations for MiniDFSCluster
- * and MiniYarn
+ * and MiniYarn.
+ *
+ * If you use this class in your project, please make sure to add a dependency to
+ * <tt>hadoop-minikdc</tt>, e.g. in your <tt>pom.xml</tt>:
+ * <pre>
+ * ...
+ * &lt;dependencies>
```

Comment:
If you do the example code as follows, you can use the `<` and make the sample more readable.
```
<pre>{@code

<dependency>
    <groupId>org.apache.hadoop</groupId>
    ...
</dependency>

}</pre>
```